List of meetings implemented

pierrecf opened the pull request:
Summary
Created a UI to display a list of one or more meetings, showcasing all meetings available.

Details
Implemented the following changes to accomplish the user story:

Implemented UI components to render the list of meetings.
Designed the layout to accommodate one or more meetings effectively.
Integrated functionality to fetch and display all meetings dynamically.

pierre-cpvd committed:
List of meetings implemented

AJN-jalaU commented on src/app/modules/app/pages/meetings/meetings.component.ts:
```diff
@@ -1,8 +1,54 @@
-export class MeetingsComponent {}
+export class MeetingsComponent implements OnInit {
+  headers: string[] = MEETING_HEADERS;
+  data: Meeting[] = [];
```
Maybe we should avoid using "data" word. Everything can be "data".
Here "data" could be meetings or something similar

AJN-jalaU commented on src/app/modules/app/pages/meetings/meetings.component.ts:
```diff
@@ -1,8 +1,54 @@
+  }
+
+  private handleSuccess(data: Meeting[]): void {
+    console.log(data);
+    this.data = data;
```
Remove Console log
Maybe we should avoid using "data" word. Everything can be "data".
Here "data" could be meetings or something similar

Adrian-Mendoza-Jala reviewed (commented):
Do you have an image of how it looks?

pierre-cpvd committed:
Merge branch 'develop' of github.com:JU-DEV-LatamBootcamp/Klockan-Frontend into feature/TG-45-list-of-meetings

pierre-cpvd committed:
Removed logs and fixed results naming

pierrecf commented:
<img width="1393" alt="Screenshot 2024-02-02 at 4 29 32 PM" src="https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/assets/151547894/f5ffc807-ed18-4e3a-8c71-9d1608d71258">
Yeah this is how it looks, i just fixed the issues metioned byt @AndresJavierNazzari

pierre-cpvd committed:
Merge branch 'develop' of github.com:JU-DEV-LatamBootcamp/Klockan-Frontend into feature/TG-45-list-of-meetings

pierre-cpvd committed:
FIxed extension of base class

pierrecf commented:
Summary
Created a UI to display a list of one or more meetings, showcasing all meetings available.

Details
Implemented the following changes to accomplish the user story:

Implemented UI components to render the list of meetings.
Designed the layout to accommodate one or more meetings effectively.
Integrated functionality to fetch and display all meetings dynamically.
<img width="1393" alt="Screenshot 2024-02-02 at 4 29 32 PM" src="https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/assets/151547894/f40194a0-f360-4b64-8d63-571590ea3970">
